crates/swc_ecma_raw_lexer/src/jsx.rs

```diff
@@ -0,0 +1,7 @@
+use logos::Logos;
+
+use crate::LogosError;
+
+#[derive(Logos, Debug, Clone, Copy, PartialEq, Eq)]
+#[logos(error = LogosError)]
+pub enum JsxToken {}
```

Comment:
`<` in jsx maybe `BinOpToken::Lt` or `JSXTagStart`, logos lexer is so hard using a kind enum to express two token kind. So I think we don't generate jsx token by logos lexer. 
We can think of logos lexer as a basic lexer that generate basic token kind such as `LtAngle` ....
Lexer using logos and state to generate more specific Token.
Do you think it a better way?

Reply:
Yeap, that's the way I was thinking of.